server/datastore/sql/index.ts

```diff
@@ -59,22 +59,47 @@ export class SqlDataStore implements Datastore {
     );
   }
 
-  getPost(id: string): Promise<Post | undefined> {
-    throw new Error('Method not implemented.');
+  async getPost(id: string): Promise<Post | undefined> {
+    // throw new Error('Method not implemented.');
+    return await this.db.get<Post>('SELECT TOP 1 From posts where postId = ?', id);
```

Comment:
`TOP 1` isn't standard SQL as far as I know, you could do `LIMIT 1`, or just omit this entirely since we know identifiers are unique, it should return exactly zero or one result.

Reply:
right.